File: EmgenexBusinessPortal/BusinessPanel/src/app/models/business-profile-model.ts
export class BusinessProfileModel {
    constructor(
        public BusinessName: string = null,
        public FilePath: string = null,
        public FullName: string = "",
        public Email: string = null,
        public Description: string = null,
        public Address: string = null,
        public About: string = null,
        public DomainUrl: string = null,
        public RelativeUrl: string = null,
        public City: string = null,
        public State: string = null,
        public Country: string = null,
        public OtherEmails: string = null,
        public Logo: string = null,
        public IsActive: boolean = false,
        public SalesGroupBy: number = null,
        public DateRange: number[] = [],
        public SalesGroup: number[] = [],
        public LogoUrl: string = null,
        public LogoPath: string = null,
    ) { }
}

File: EmgenexBusinessPortal/Models/BusinessProfileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EmgenexBusinessPortal.Models
{
    public class BusinessProfileModel
    {
        public int Id { get; set; }
        public string BusinessName { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public string Logo { get; set; }
        public string Country { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public string About { get; set; }
        public string RelativeUrl { get; set; }
        public string DomainUrl { get; set; }
        public bool IsActive { get; set; }
        public int? Status { get; set; }
        public int CreatedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public string CreatedUser { get; set; }
        public string Updateduser { get; set; }
        public int? DateRange { get; set; }
        public int? SalesGroup { get; set; }
        public string OtherEmails { get; set; }
    }
}